src/api/helpers.py

```diff
@@ -130,3 +131,21 @@ def create_page(
     )
 
     return page_output
+
+
+def get_month_range(year_month: str) -> tuple[date, date]:
+    """
+    Parse month in `YYYY-MM` format.
+    Return two dates for first day and last day of month.
+    """
+    year, month = map(int, year_month.split("-"))
```

Comment:
do we need to add extra validation or exception handling here?

Reply:
I see now `Query(pattern=r"\d{4}-\d{2}")` below so that would make this safe, still wondering if that validation should be done here though?

Reply:
Maybe we can go EAFP style by trying to construct a date object and catch the exception?

See also: http://localhost:8000/tips/lbyl-vs-eafp-style